seed/static/seed/js/controllers/export_inventory_modal_controller.js

```diff
@@ -30,27 +32,40 @@ angular.module('BE.seed.controller.export_inventory_modal', []).controller('expo
     $scope.include_label_header = false;
     $scope.inventory_type = inventory_type;
     $scope.exporting = false
+    $scope.exporter_progress = {
+      progress: 0,
+      status_message: '',
+    };
 
     $scope.export_selected = function (export_type) {
       var filename = $scope.export_name;
       var ext = '.' + export_type;
       if (!_.endsWith(filename, ext)) filename += ext;
-      spinner_utility.show()
       $scope.exporting = true
-      return $http.post('/api/v3/tax_lot_properties/export/', {
-        ids: ids,
-        filename: filename,
-        profile_id: profile_id,
-        export_type: export_type,
-        include_notes: $scope.include_notes
-      }, {
+      $http.get('/api/v3/tax_lot_properties/start_export', {}, {
         params: {
           organization_id: user_service.get_organization().id,
-          inventory_type: inventory_type
-        },
-        responseType: export_type === 'xlsx' ? 'arraybuffer' : undefined
+        }
+      }).then(data => {
+        uploader_service.check_progress_loop(data.data.progress_key, 0, 1, 
+          function () { }, 
+          function () { }, 
+          $scope.exporter_progress)
+        return $http.post('/api/v3/tax_lot_properties/export/', {
+          ids: ids,
+          filename: filename,
+          profile_id: profile_id,
+          export_type: export_type,
+          include_notes: $scope.include_notes,
+          progress_key: data.data.progress_key
+        }, {
+          params: {
+            organization_id: user_service.get_organization().id,
+            inventory_type: inventory_type
+          },
+          responseType: export_type === 'xlsx' ? 'arraybuffer' : undefined
+        })
```

Comment:
hit backend for a new `ProgressData` object, use that returned `progress_key` to `check_progress_loop` and `tax_lot_properties/export`